configs/supertokens.json

```
@@ -84,7 +205,7 @@
     ]
   },
   "js_render": true,
-  "js_wait": 2,
+  "js_wait": 8,
```

```suggestion
  "js_wait": 5,
```

A high number could lead into some timeouts, we recommend using 5 as a max value but if it didn't worked with 5, we can try it out.